src/Core/Models/Api/Request/SsoConfigRequestModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Bit.Core.Models.Api
```

Comment:
I don't think we will be exposing this in the API like other componenets. This will be configed from enterprise portal, which will just have CRUD view models.